kayenta-objectstore-configbin/src/main/java/com/netflix/kayenta/configbin/service/ConfigBinRemoteService.java

```diff
@@ -21,22 +21,26 @@
 import retrofit.http.*;
 
 public interface ConfigBinRemoteService {
-  @GET("/{ownerApp}/{configType}")
-  String list(@Path("ownerApp") String ownerApp, @Path("configType") String configType);
 
-  @DELETE("/{ownerApp}/{configType}/{configName}")
+  @GET("/browse/{ownerApp}/{configType}?pageSize=1000&showDeleted=false")
+  String list(
+      @Path("ownerApp") String ownerApp,
+      @Path("configType") String configType,
+      @Query("nextPageId") String nextPageId);
```

Comment:
API requires paging now

Reply:
Does this require a coordinated change from the UI then?

Reply:
Nevermind -- I see this is used by us internally rather than this being what Kayenta exposes to Spinnaker.

Reply:
No change on the UI side. At start Kayenta will query for all configurations stored in ConfigBin. In this case we will pull the first 1000. We typically have 400 active configurations. But, in the event that we have over 1000, the ConfigBinStorageService will pull the next 1000 and combine results until no more pages are returned.